src/python/pants/backend/terraform/lint/tfsec/tfsec_integration_test.py

```diff
@@ -31,8 +31,9 @@ def test_run_tfsec():
 
     rule_runner.set_options(
         [
-            "--terraform-tfsec-args='--no-colour'",
+            "--terraform-tfsec-args='--no-color'",
```

Comment:
lovely gem from the help text:
```
      --no-color                     Disable colored output (American style!)
      --no-colour                    Disable coloured output
```

Reply:
Haha I saw that. I accidentally changed that line while I was refactoring, and then realized I probably unintentionally looked like an ass :P